dependencies.props

```diff
@@ -21,15 +21,15 @@
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-04</CoreFxExpectedPrerelease>
-    <CoreClrExpectedPrerelease>preview2-25316-03</CoreClrExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview1-25320-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>preview1-25319-05</CoreClrExpectedPrerelease>
```

Comment:
See my fix with https://github.com/dotnet/corefx/pull/20025/commits/444ea73cdd596059f2ef5ed4ba4a46ae6819d95e. The issue was we were downgrading the package minor version so it was picking an older runtime package without these changes.